examples/heatmap.py

```
@@ -20,6 +20,7 @@ class HeatMap(Campaign):
         p.ParamCmdLineArg("heat", "xprocs", 2, [4]),
         p.ParamCmdLineArg("heat", "yprocs", 3, [3]),
         p.ParamRunner("heat", "nprocs", [12]),
+        p.ParamAdiosXMLArg("heat", "<adios_transform> {'xml':'heat_transfer.xml','adios-group':'heat','var':'T'}", ['zfp:accuracy=.001', 'sz']),
```

This should raise an error:
```
p.ParamAdiosXML("heat", "adios_transform:heat_transfer.xml:heat:T",
                        ["zfp:accuracy=.001", "sz"]),
p.ParamAdiosXML("heat", "adios_transform:heat_transfer.xml:heat:T",
                        ["zfp:accuracy=.1"]),
```
(should be moved into one param entry),
but this should be allowed:
```
p.ParamAdiosXML("heat", "adios_transform:heat_transfer.xml:heat:T",
                        ["zfp:accuracy=.001", "sz"]),
p.ParamAdiosXML("heat", "adios_transform:heat_transfer.xml:heat:dT",
                        ["zfp:accuracy=.1"]),
```
since it's specifying a different variable to transform.